src/xlat-i/xi-Entry.C

```diff
@@ -2419,34 +2487,34 @@ void Entry::setAccelParam(ParamList* apl) { accelParam = apl; }
 void Entry::setAccelCodeBody(XStr* acb) { accelCodeBody = acb; }
 void Entry::setAccelCallbackName(XStr* acbn) { accelCallbackName = acbn; }
 
-int Entry::isThreaded(void) { return (attribs & STHREADED); }
-int Entry::isSync(void) { return (attribs & SSYNC); }
-int Entry::isIget(void) { return (attribs & SIGET); }
+int Entry::isThreaded(void) { return (hasAttribute(STHREADED)); }
+int Entry::isSync(void) { return (hasAttribute(SSYNC)); }
+int Entry::isIget(void) { return (hasAttribute(SIGET)); }
 int Entry::isConstructor(void) {
   return !strcmp(name, container->baseName(0).get_string());
 }
-bool Entry::isMigrationConstructor() { return isConstructor() && (attribs & SMIGRATE); }
-int Entry::isExclusive(void) { return (attribs & SLOCKED); }
-int Entry::isImmediate(void) { return (attribs & SIMMEDIATE); }
-int Entry::isSkipscheduler(void) { return (attribs & SSKIPSCHED); }
-int Entry::isInline(void) { return attribs & SINLINE; }
-int Entry::isLocal(void) { return attribs & SLOCAL; }
-int Entry::isCreate(void) { return (attribs & SCREATEHERE) || (attribs & SCREATEHOME); }
-int Entry::isCreateHome(void) { return (attribs & SCREATEHOME); }
-int Entry::isCreateHere(void) { return (attribs & SCREATEHERE); }
-int Entry::isPython(void) { return (attribs & SPYTHON); }
-int Entry::isNoTrace(void) { return (attribs & SNOTRACE); }
-int Entry::isAppWork(void) { return (attribs & SAPPWORK); }
-int Entry::isNoKeep(void) { return (attribs & SNOKEEP); }
+bool Entry::isMigrationConstructor() { return isConstructor() && hasAttribute(SMIGRATE); }
+int Entry::isExclusive(void) { return (hasAttribute(SLOCKED)); }
+int Entry::isImmediate(void) { return (hasAttribute(SIMMEDIATE)); }
+int Entry::isSkipscheduler(void) { return (hasAttribute(SSKIPSCHED)); }
+int Entry::isInline(void) { return hasAttribute(SINLINE); }
+int Entry::isLocal(void) { return hasAttribute(SLOCAL); }
+int Entry::isCreate(void) { return (hasAttribute(SCREATEHERE)) || (hasAttribute(SCREATEHOME)); }
+int Entry::isCreateHome(void) { return (hasAttribute(SCREATEHOME)); }
+int Entry::isCreateHere(void) { return (hasAttribute(SCREATEHERE)); }
+int Entry::isPython(void) { return (hasAttribute(SPYTHON)); }
+int Entry::isNoTrace(void) { return (hasAttribute(SNOTRACE)); }
+int Entry::isAppWork(void) { return (hasAttribute(SAPPWORK)); }
+int Entry::isNoKeep(void) { return (hasAttribute(SNOKEEP)); }
 int Entry::isSdag(void) { return (sdagCon != 0); }
-bool Entry::isTramTarget(void) { return (attribs & SAGGREGATE) != 0; }
-int Entry::isWhenIdle(void) { return attribs & (SWHENIDLE ^ SLOCAL); }
```

Comment:
Is it ok to remove the `SLOCAL` flag here?

Reply:
Yes, I confirmed with Justin that the way the flags are set allows not checking SLOCAL there(there is a change made elsewhere to make this behavior correct).